fiftyone/plugins/core.py

```diff
@@ -564,18 +589,20 @@ def _iter_plugin_metadata_files():
 
 
 def _get_plugin(name, enabled=None, check_for_duplicates=True):
+    """Find plugin packages matching name."""
```

Comment:
I think it is confusing for `_get_plugin(name)` to always be a generator. The default arg of  `check_for_duplicates=True` implies that it will either return a single plugin or raise an error if there are duplicates.

A solution would be to add an optional syntax that returns a list rather than a single plugin:

```py
# Returns a list of one or more plugins
plugins = _get_plugin(name, include_duplicates=True)
```

Reply:
I’m aware. Comment still stands